src/sentry/deletions/defaults/group.py

```diff
@@ -26,6 +28,8 @@
 GROUP_CHUNK_SIZE = 100
 EVENT_CHUNK_SIZE = 10000
 GROUP_HASH_ITERATIONS = 10000
+# Batch size for nullifying group_hash_metadata.seer_matched_grouphash_id references to avoid database timeouts
+GROUP_HASH_METADATA_BATCH_SIZE = 10
```

Comment:
We're seeing 100 hashes updates taking over 30 seconds long, thus, this should be good enough.